Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -291,4 +316,53 @@ - (NSString *)formatProducts:(NSDictionary *)obj
     return [output componentsJoinedByString:@";"];
 }
 
+///-------------------------
+/// @name Video Tracking
+///-------------------------
+
+-(void)trackHeartbeatEvents:(SEGTrackPayload *)payload
+{
+    if ([payload.event isEqualToString:@"Video Playback Started"]){
+        self.config = [self createConfig:payload];
+       self.ADBMediaHeartbeat = [self initializeMediaHeartbeat:self.config andDelegate:nil];
```

Comment:
looks like a formatting error? please run the formatter.

Reply:
Good catch — looks like something happened and the formatter was removed. Running